### PR TITLE
Maven emr

### PR DESCRIPTION
**Pull request by lucky-suman**
Remove application "MxNet" for the Maven ecosystem.

### CODE REVIEW

File: rudra/deployments/emr_scripts/maven_emr.py [hunk 1]
```diff
@@ -23,9 +23,11 @@ def run_job(self, input_dict):
         log_file_name = '{}.log'.format(name)
 
         log_uri = 's3://{bucket}/{log_file}'.format(
-            bucket='{}-automated-analytics-spark-jobs'.format(self.env),
+            bucket=self.bucket_name,
             log_file=log_file_name)
 
+        logger.info("Logs are gonna store at {}".format(log_uri))
```
Comment (rootAvish) [inline rudra/deployments/emr_scripts/maven_emr.py]: Minor nitpick, this is grammatically incorrect 🙃, please change it to "logs will be stored at".

File: scripts/bootstrap_maven.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+sed -i '/export DEEPLEARNING_WORKERS_COUNT=/d' ~/.bashrc
+echo "export DEEPLEARNING_WORKERS_COUNT="$worker_count >> ~/.bashrc
+
+source ~/.bashrc
+
+echo $DEEPLEARNING_WORKERS_COUNT
+echo $DEEPLEARNING_WORKERS_PATH
+
+echo "Environment variables are set!"
```
Comment (rootAvish) [inline scripts/bootstrap_maven.sh]: You can remove everything before this line and all would still work fine. I suggest you do that now that we are splitting for different ecosystems.

Comment (rootAvish) [reply scripts/bootstrap_maven.sh]: I guess not everything, please retain the shebang 😉 (`#!/bin/sh`)

File: rudra/deployments/emr_scripts/maven_emr.py (hunk 1 above)
Comment (lucky-suman) [reply rudra/deployments/emr_scripts/maven_emr.py]: ahh Got it !

File: scripts/bootstrap_maven.sh (hunk 2 above)
Comment (lucky-suman) [reply scripts/bootstrap_maven.sh]: Ok avishkar will do it!

**Pull request merged**